plugins/woocommerce/src/Admin/API/Reports/Orders/DataStore.php

```diff
@@ -534,6 +546,52 @@ protected function get_coupons_by_order_ids( $order_ids ) {
 		return $coupons;
 	}
 
+	/**
+	 * Get order attributions data from order IDs.
+	 *
+	 * @param array $order_ids Array of order IDs.
+	 * @return array
+	 */
+	protected function get_order_attributions_by_order_ids( $order_ids ) {
+		global $wpdb;
+		$order_meta_table   = OrdersTableDataStore::get_meta_table_name();
+		$included_order_ids = implode( ',', $order_ids );
```

Comment:
🚧 We should sanitize the IDs
```suggestion
		$included_order_ids = implode( ',', array_map( 'absint', $order_ids ) );
```

Reply:
Thanks! I actually got the `implode( ',', $order_ids );` from the other functions (e.g. `get_products_by_order_ids`). 

Anyway, I have applied your suggestion. Good to be defensive here.